gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -319,6 +319,10 @@ gst_tensor_converter_class_init (GstTensorConverterClass * klass)
     g_strfreev (str_array);
   }
 
+  /* append caps string for flexible tensors */
+  gst_caps_append (pad_caps,
+      gst_caps_from_string (GST_TENSORS_FLEX_CAP_DEFAULT));
```

Comment:
Can you do this with the style of LINE 302-206 if this capability is built-in?

Reply:
updated